manager.go

```diff
@@ -609,6 +620,15 @@ func (m *Manager) InitWithOptions(elf io.ReaderAt, options Options) error {
 		return err
 	}
 
+	if m.options.AdditionalExcludedFunctionCollector != nil {
+		for key, prog := range m.collectionSpec.Programs {
+			if prog.AttachTo != "" && m.options.AdditionalExcludedFunctionCollector.ShouldExcludeFunction(key, prog) {
```

Comment:
I'm curious why you are filtering to only programs with `AttachTo` set? Is that specific to your use case, or would all users of this want that?

Reply:
ah yes sorry in my first version I was only passing the attachto, but now that I'm passing the whole prog it makes less sense